api_tests/src/comment.spec.ts

```diff
@@ -763,10 +766,16 @@ test("Dont send a comment reply to a blocked community", async () => {
   // Alpha creates a comment
   let commentRes = await createComment(alpha, alphaPost.post.id);
   expect(commentRes.comment_view.comment.content).toBeDefined();
-  await resolveComment(beta, commentRes.comment_view.comment);
+  let alphaComment = await resolveComment(
+    beta,
+    commentRes.comment_view.comment,
+  );
+  if (!alphaComment) {
+    throw "Missing alpha comment before block";
+  }
```

Comment:
I would use simple expect instead of such messages because its shorter and simpler, but oh well